packages/react/src/analytics/integrations/AnalyticsApi/AnalyticsApi.ts

```diff
@@ -0,0 +1,107 @@
+import {
+  type AnalyticsApiIntegrationOptions,
+  type AnalyticsApiTrackData,
+} from './types/types.js';
+import { client } from '@farfetch/blackout-client';
+import {
+  type EventData,
+  EventTypes,
+  integrations,
+  type LoadIntegrationEventData,
+  PageTypes,
+  type StrippedDownAnalytics,
+  type TrackTypesValues,
+  utils,
+} from '@farfetch/blackout-analytics';
+import { isArray } from 'lodash';
```

Comment:
Do not use lodash, you need to use `lodash-es` always since this package is ESM-only now.

```suggestion
import { isArray } from 'lodash-es';
```